### ISSUE TITLE
Password Requirements

### ISSUE THREAD

**Issue by lorenzschmid** [standardfile/standardfile.github.io#2]
As mentioned in [this issue on the ruby-server](https://github.com/standardfile/ruby-server/issues/25) there should be a general requirement mentioned in the specification on the user's password enforced by the app developer:

- Enforce a minimum password length
- Indicate that a password reset is impossible
- Ask the user to confirm the password

I am unsure though which site suits best (API specs or dev guide).

**Comment by mobitar** [comment]
Yes, the web client needs to do a better job at that. The iOS app does it pretty well. Confirms and communicates. This is definitely on the agenda.

**Comment by mobitar** [comment]
I got confused by the repos. You're talking about communicating it in the docs. True. I think that should go in the client development guide.

**Comment by lorenzschmid** [comment]
Sorry for the confusion. As you understood correctly I meant a general indication for any app using the standard file backend. I will try to add that.
